docs/development/pigs/pig-013.rst

```diff
@@ -239,6 +239,10 @@ However, we suggest to stick to ``gammapy`` as a package with a very small set
 of dependencies, and then use the environment specifications to create the stack
 and execution environment that most users want, not metapackages for now.
 
+TODO: now I'm having second thoughts. Maybe we should go the pip & conda metapackages route?
```

Comment:
doesn't @astrofrog have similar setup for glue? Maybe experiences from there could help with the decision for gammapy.